### DISCUSSION TITLE
thanhle.blog/minh-da-viet-mot-tuan-1-blog-trong-vong-1-nam

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#66] [Ideas]
# thanhle.blog/minh-da-viet-mot-tuan-1-blog-trong-vong-1-nam

Xem coi 1 năm thằng Thành này viết blog ra hồn không

https://thanhle.blog/blog/minh-da-viet-mot-tuan-1-blog-trong-vong-1-nam

**Comment by thanhlmm** [comment]
Form câu hỏi để chúng ta hỉu nhau hơn 😍
https://j8gflmjpy7w.typeform.com/to/acsK53WC

**Comment by chungnq253** [comment]
Vẫn luôn ủng hộ và hóng blog của Thành mỗi tuần. Nội dung thân thuộc và rất có value 😙

**Reply by thanhlmm** [reply]
thanks bro!

**Comment by nvh95** [comment]
Mình mới biết đến blog của bạn và thấy các mối quan tâm về web của bạn cũng tương tự mình. Hi vọng mình sẽ học được nhiều từ bạn Chúc bạn giữ được niềm vui khi viết blog trong thời gian sắp tới ^^
(Mình cũng mới subscribe blog và follow Twitter bạn nhé 💪)

**Reply by thanhlmm** [reply]
Haha. Thank you pro, nhớ chém t khi t viết gì đó sai sai nhé!

**Reply by thanhlmm** [reply]
Hey bro có chạy Techcombank sắp tới không? 🏃‍♂️

**Reply by nvh95** [reply]
Mình có. Mình chạy cự ly HM ở giải đó ☺️ 🏃‍♂️

**Reply by thanhlmm** [reply]
T chạy FM 😅. T định rủ đi bro uống cf sau buổi đó mà theo kinh nghiệm mấy lần trước thì lần này chạy xong t cũng toang :)))

FB của mình https://www.facebook.com/thanh.lmm/
Khi nào mời bạn đi cf nói chuyện open source dc ko?
